packages/mobile/package.json

```diff
@@ -208,6 +208,7 @@
     "@types/uuid": "^8.3.0",
     "@typescript-eslint/eslint-plugin": "^4.25.0",
     "@valora/eslint-config-typescript": "^0.0.1",
+    "@valora/resolve-kit": "^0.0.3-dev",
```

Comment:
I will update this once we released the lastest changes for resolve-kit: https://github.com/valora-inc/resolve-kit/pull/9